### ISSUE TITLE
UrbanAirship.addCustomEvent throwing errors in iOS

### ISSUE THREAD

**Issue by jacquesdev** [urbanairship/react-native-module#329]
❗For how-to inquiries involving Airship functionality or use cases, please
contact (support)[https://support.airship.com/].

# Preliminary Info

### What Airship dependencies are you using?

React-native SDK 11.0.1

# Report

### What unexpected behavior are you seeing?

```
import {
  UrbanAirship,
  CustomEvent,
} from 'urbanairship-react-native'

const event = new CustomEvent(eventName)
  for (const [key, property] of Object.entries(properties)) {
    event.addProperty(key, property)
  }

UrbanAirship.addCustomEvent(event)
```

UrbanAirship.addCustomEvent(event) is throwing `No action found with name add_custom_event_action, skipping action` in iOS.

### What is the expected behavior?
To send a custom event as expected

**Comment by rlepinski** [comment]
Thanks for the report, we will take a look

**Comment by Ulrico972** [comment]
Hello @jacquesdev,

I wasn't able to reproduce the issue on my side. Indeed, the ```add_custom_event_action``` is a default action in the SDK. It is defined in our UADefaultActions.plist and registered at the take off.

I would like to try to reproduce with the exact Airship SDK versions you are using. Could you tell me how did you implement it ? Did you use a static pod, a framework pod or something else ?

Also, I would like to verify if the UADefaultActions.plist is present where it should be, in the built app bundle, or eventually where is it actually.
So could you please send us the content of the built app bundle ? You should be able to see it by right clicking it and select "show package contents".

**Comment by jacquesdev** [comment]
Hi @Ulrico972.

I have managed to solve this, the UADefaultActions.plist is indeed there and working as expected. The issue was that the airship.plist file was not referenced correctly in xcode, and so take off was never happening. I usually don't need to confirm things like this in the xcode debug console, since for all other implementations I've seen - the sdk issues are usually bubbled up to the react-native metro server, and so it's pretty obvious if something like this is not working as expected.

Once I knew to check the xcode logs, when I referenced the file properly it seems to now work.

Thanks for investigating this for us, but we can now close this one
